### PR TITLE
OSSMDOC-595: service mesh 2.1.3 z stream release notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-595

Link to 2.1.3 doc preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-595/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-1-3

Link to 1.1.19 doc preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-595/service_mesh/v1x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-1-1-19 

Additional information:

### CODE REVIEW

**Review by jwendell — APPROVED** [review]

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -122,10 +122,10 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
 :SMProductShortName: Service Mesh
-:SMProductVersion: 2.1.2.1
+:SMProductVersion: 2.1.3
 :MaistraVersion: 2.1
 //Service Mesh v1
-:SMProductVersion1x: 1.1.18.1
+:SMProductVersion1x: 1.1.19
```
Comment (pbajjuri20) [inline _attributes/common-attributes.adoc:128]: @rh-tokeefe: The version for SM 1.1 is 1.1.18-2.

FYI: 1.1 Errata name: Red Hat OpenShift Service Mesh 1.1.18.2 Containers

File: modules/ossm-rn-new-features-1x.adoc [hunk 2]
```diff
@@ -19,6 +19,28 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 1.1.19
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features-1x.adoc:22]: The version for SM 1.1 is 1.1.18-2.

FYI: 1.1 Errata name for 2.1.3 release: Red Hat OpenShift Service Mesh 1.1.18.2 Containers

File: modules/ossm-rn-known-issues.adoc [hunk 3]
```diff
@@ -30,6 +30,9 @@ These limitations exist in {SMProductName}:
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-1211[OSSM-1211]
+OpenShift Service Mesh Federation failover Transport Layer Security (TLS) error when setting `importAsLocal` to `true` in two separate federated meshes.
```
Comment (JStickler) [inline modules/ossm-rn-known-issues.adoc:34]: I think we need to be clearer here, the problem is not so much that there is an error message as that Failover doesn't work.  Maybe something like, "Configuring Federated service meshes for failover does not work as expected.  There is no workaround at this time."

**Review by JStickler — CHANGES_REQUESTED** [review]

File: _attributes/common-attributes.adoc [hunk 4]
```diff
@@ -122,10 +122,10 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
 :SMProductShortName: Service Mesh
-:SMProductVersion: 2.1.2.1
+:SMProductVersion: 2.1.3
 :MaistraVersion: 2.1
 //Service Mesh v1
-:SMProductVersion1x: 1.1.18.1
+:SMProductVersion1x: 1.1.18-2
```
Comment (jshaughn) [inline _attributes/common-attributes.adoc:128]: is it supposed to be `1.1.18.2` ?  I don't know the answer but  I see above that it was `2.1.2.1`.

**Review by jshaughn — COMMENTED** [review]
Kiali versions look good.  A couple of comments re: the 1.1 version.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**